testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -66,6 +71,15 @@ use crate::run_loop::nakamoto::{Globals, RunLoop};
 use crate::run_loop::RegisteredKey;
 use crate::BitcoinRegtestController;
 
+#[cfg(test)]
+lazy_static! {
+    /// Mutex to stall the relayer thread right before it creates a miner thread.
+    pub static ref TEST_MINER_THREAD_STALL: TestFlag<bool> = TestFlag::default();
+
+    /// Mutex to stall the miner thread right after it starts up (does not block the relayer thread)
+    pub static ref TEST_MINER_THREAD_START_STALL: TestFlag<bool> = TestFlag::default();
+}
```

Comment:
```suggestion
/// Mutex to stall the relayer thread right before it creates a miner thread.
 #[cfg(test)]
 pub static TEST_MINER_THREAD_STALL: LazyLock<TestFlag<bool>> = LazyLock::new(TestFlag::default);

 /// Mutex to stall the miner thread right after it starts up (does not block the relayer thread)
 #[cfg(test)]
 pub static TEST_MINER_THREAD_START_STALL: LazyLock<TestFlag<bool>> =
     LazyLock::new(TestFlag::default);
```

Reply:
I had this originally, but it seems that merging `develop` overrode it.